src/run_generator.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, TemplateData=None, RunList=None):
                 'injectors': injectors,
                 'java': template["java"],
                 'jar': template["jar"],
+                'tag': run["args"]["Tag"],
```

Comment:
Should avoid making changes to the core in `dev-gui` without getting them pulled into `dev`.

Reply:
Yeah I agree

Reply:
#148 addresses this.